app/views/images/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="album py-5 bg-light">
+  <div class="container">
+
+    <div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-3">
+      <% @images.each do |image| %>
+      <div class="col card-img">
+        <a href="images/<%= image.id %>">
+          <div class="card shadow-sm">
+            <% if image.pic.attached? %>
+            <%= image_tag(image.pic) %>
+            <% end %>
+            <div class="card-body">
+              <i class="fas fa-comment"></i> <i class="fas fa-heart"></i> <i
+                class="fas fa-external-link-square-alt"></i>
+            </div>
+          </div>
+        </a>
+      </div>
+      <% end %>
+    </div>
+  </div>
```

Comment:
I'm guessing this is to make sure all pictures are the same size? (or don't go past a certain maximum size)?